repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -425,36 +442,52 @@ def gather_target_repositories(context, indata):
+                ' the current subscription is entitled to install the requested target version {version}.'
+                ' If you used the --no-rhsm option (or the LEAPP_NO_RHSM=1 environment variable is set),'
+                ' ensure the custom repository file is provided with'
+                ' properly defined repositories and that the --enablerepo option for leapp is set if the'
+                ' repositories are defined in any repofiles under the /etc/yum.repos.d/ directory.'
+                ' For more information on custom repository files, see the documentation.'
+                ' Finally, verify that the "/etc/leapp/files/repomap.csv" file is up-to-date.'
+            ).format(version=api.current_actor().configuration.version.target)),
+            reporting.RelatedResource("file", "/etc/leapp/files/repomap.csv"),
+            reporting.RelatedResource("file", "/etc/yum.repos.d/")
+        ])
+        raise StopActorExecution()
     if missing_custom_repoids:
-        raise StopActorExecutionError(
-            message='Some required custom target repositories are not available.',
-            details={'hint': (
-                ' The most probably you are using custom or third party actor'
-                ' that produces CustomTargetRepository message or you did a typo'
-                ' in one of repoids specified on command line for the leapp --enablerepo'
-                ' option.'
-                ' Inside the upgrade container, we are not able to find such'
-                ' repository inside any repository file. Consider use of the'
-                ' custom repository file regarding the official upgrade'
-                ' documentation or check whether you did not do a typo in any'
-                ' repoids you specified for the --enablerepo option of leapp.'
-                )
-            }
-        )
-
+        reporting.create_report([
+            reporting.Title('Some required custom target repositories have not been found'),
+            reporting.Summary(
+                'During the configuration for the target system, one or more repository IDs were missing,'
+                ' have been identified. This can happen when a repository ID was entered incorrectly either'
+                ' while using the --enablerepo option of leapp, or in a third party actor that produces a'
+                ' CustomTargetRepositoryMessage.\n'
+                'The following repositories IDs could not be found in the target configuration:\n'
+                '- {}\n'.format('\n- '.join(missing_custom_repoids))
+            ),
+            reporting.Tags([reporting.Tags.REPOSITORY]),
+            reporting.Flags([reporting.Flags.INHIBITOR]),
+            reporting.Remediation(hint=(
+                'Consider using the custom repository file, which is the official'
+                ' upgrade documentation. Check whether a repository ID has been'
```

Comment:
the wording feels to me like other way than the custom repository file is not official. if you know what I mean. but both ways (--enablerepo & the custom repofile) are equal from our POV.

Reply:
and again, missing link to the official documentation.

Reply:
It has been late last night when I applied the changes, yeah this text isn't saying what it was supposed to say.